### PR TITLE
Added validations for k8s gateway

### PR DESCRIPTION
**Pull request by josunect**
- Added validations documentation for k8s Gateway (1501, 1502, GWAPI)
- Updated color icon 

https://deploy-preview-618--kiali.netlify.app/docs/features/validations/

REF https://github.com/kiali/kiali/issues/5501 https://github.com/kiali/kiali/pull/5687

### CODE REVIEW

**Review by hhovsepy — APPROVED** [review]
lgtm, will wait for Jay's review.

**Comment by josunect** [comment]
validation is failing, but it looks like https://v1-60.kiali.io is not deployed

File: content/en/docs/Features/validations.md [hunk 1]
```diff
@@ -999,5 +999,59 @@ Choose existing and accessible namespace to export to.
...
 #### Severity
 
-<i class="fas fa-times-circle"></i> Error
+<i class="fas fa-times-circle text-danger"></i> Error
 
+## K8s Gateway {#k8sgateways}
+
+### KIA1501 - More than one K8s Gateway for the same host and port combination 
+
+A k8s Gateway define a point where traffic can be translated to Services within the cluster. This is defined through the listeners or addresses. This validation warns the scenario where there is more than one Listener for the same port and host combinations in different k8s Gateways, where the traffic handling can be in a conflict.
```
Comment (jshaughn) [inline content/en/docs/Features/validations.md]:
```suggestion
A k8s Gateway defines a point where traffic can be translated to Services, within the cluster. This is defined through listeners or addresses. This validation warns when finding multiple Listener definitions for the same port and host combination, in different k8s Gateways.  In this case the traffic handling can be in conflict.
```

File: content/en/docs/Features/validations.md [hunk 2]
```diff
@@ -999,5 +999,59 @@ Choose existing and accessible namespace to export to.
...
+
+### KIA1501 - More than one K8s Gateway for the same host and port combination 
+
+A k8s Gateway define a point where traffic can be translated to Services within the cluster. This is defined through the listeners or addresses. This validation warns the scenario where there is more than one Listener for the same port and host combinations in different k8s Gateways, where the traffic handling can be in a conflict. 
+
+The exception to this rule is where the listeners specify different handlers. That is the reason why the severity is a warning. 
+
+#### Resolution
+
+Remove the duplicate k8s gateway entries or merge the two k8s gateway definitions into a single one.
```
Comment (jshaughn) [inline content/en/docs/Features/validations.md]:
```suggestion
Remove or merge the duplicate k8s gateway entries.
```

File: content/en/docs/Features/validations.md [hunk 3]
```diff
@@ -999,5 +999,59 @@ Choose existing and accessible namespace to export to.
...
+
+#### Example
+
+```yaml
+{{% readfile file="/static/files/validation_examples/1501.yaml" %}}
+```
+
+### KIA1502 - More than one K8s Gateway for the address and type combination
+
+A k8s Gateway define a point where traffic can be translated to Services within the cluster. This is defined through the listeners or addresses. This validation warns the scenario where there is more than one address (type and value) in different k8s Gateways, where the traffic handling can be in a conflict.
```
Comment (jshaughn) [inline content/en/docs/Features/validations.md]:
```suggestion
A k8s Gateway defines a point where traffic can be translated to Services within the cluster. This is defined through listeners or addresses. This validation warns when finding more than one address (type and value) in different k8s Gateways, where the traffic handling can be in conflict.
```

File: content/en/docs/Features/validations.md [hunk 4]
```diff
@@ -999,5 +999,59 @@ Choose existing and accessible namespace to export to.
...
+
+#### Resolution
+
+Remove the duplicate k8s gateway entries or merge the two k8s gateway definitions into a single one.
```
Comment (jshaughn) [inline content/en/docs/Features/validations.md]:
```suggestion
Remove or merge the duplicate k8s gateway entries.
```

File: content/en/docs/Features/validations.md [hunk 5]
```diff
@@ -999,5 +999,59 @@ Choose existing and accessible namespace to export to.
...
+#### Example
+
+```yaml
+{{% readfile file="/static/files/validation_examples/1502.yaml" %}}
+```
+
+### GWAPI - Gateway API status
+
+The Gateway object provides a GatewayStatus to provide the status relative to the state represented in the spec. 
+The validations under GWAPI rule are generic and will highlight any invalid status created by the Gateway.
```
Comment (jshaughn) [inline content/en/docs/Features/validations.md]:
```suggestion
The validations under the GWAPI rule are generic and will highlight any invalid status created by the Gateway.
```

**Review by jshaughn — CHANGES_REQUESTED** [review]

**Comment by josunect** [comment]
Thanks for your review!

**Pull request merged**